scripts/zones/Beadeaux/Zone.lua

```diff
@@ -50,17 +57,66 @@ zone_object.onConquestUpdate = function(zone, updatetype)
 end
 
 zone_object.onRegionEnter = function(player, region)
-    if region:GetRegionID() <= 6 then
-        if not player:hasStatusEffect(xi.effect.CURSE_I) and not player:hasStatusEffect(xi.effect.SILENCE) then
-            player:addStatusEffect(xi.effect.CURSE_I, 50, 0, 300)
-            if player:getQuestStatus(xi.quest.log_id.BASTOK, xi.quest.id.bastok.THE_CURSE_COLLECTOR) == QUEST_ACCEPTED and player:getCharVar("cCollectCurse") == 0 then
-                player:setCharVar("cCollectCurse", 1)
+    local regionID = region:GetRegionID()
+    local pos      = player:getYPos()
```

Comment:
Can we change this var name to yPos to be more specific?  This happens below as well.